tests/pipelines/animatediff/test_animatediff_video2video.py

```diff
@@ -28,7 +28,7 @@ def to_np(tensor):
     return tensor
 
 
-class AnimateDiffVideoToVideoPipelineFastTests(PipelineTesterMixin, unittest.TestCase):
+class AnimateDiffVideoToVideoPipelineFastTests(IPAdapterTesterMixin, PipelineTesterMixin, unittest.TestCase):
```

Comment:
This pipeline will fail tests because it uses the old logic for IPAdapters with `encode_image()` instead of `prepare_ip_adapter_image_embeds`. Can I fix the IP Adapter support here? Or should it be done in a different PR?

Reply:
I think it's easier to fix it here:)